src/fluree/db/ledger/txgroup/txgroup_proto.clj

```diff
@@ -236,7 +236,7 @@ or this server is not responsible for this ledger, will return false. Else true
 (defn new-ledger-async
   "Registers new network to be created by leader."
   [group network ledger-id cmd-id signed-cmd]
-  (let [command [:new-db network ledger-id cmd-id signed-cmd]]
+  (let [command [:new-ledger network ledger-id cmd-id signed-cmd]]
     (-new-entry-async group command)))
 
 (defn find-all-dbs-to-initialize
```

Comment:
I _think_ this might need changing too, but again I could go either way. 

I guess I think of dbs as static and unchanging, while I think of ledgers as variable. Since a db is just a value, it can't be initialized; it just is. It seems to me like the ledger is the thing you're initializing, but this is a feebly held opinion.

Reply:
Agreed. Same thoughts here too. :)